database-service/tests/fixtures-generator/json2sql.py

```diff
@@ -29,8 +22,29 @@ def main(input, migration_name, output_folder, write_mode):
                 output_file.write(
                     f'INSERT INTO public.{table} ({columns}) VALUES ({values});\n')
 
-    with open(down_migration_file, write_mode) as output_file:
-        output_file.write(f'DELETE FROM public.{table}')
+
+def generate_down_migration(json_data, output_folder, migration_name, write_mode):
+    migration_file = os.path.join(output_folder, migration_name + '.down.sql')
+    with open(migration_file, write_mode) as output_file:
+        for table, data in reversed(json_data.items()):
+            for data_item in reversed(data):
+                condition = []
+                for col, val in reversed(data_item.items()):
+                    condition.append(f'{col}=\'{val}\'')
+                where = ' AND '.join(condition)
+                output_file.write(f'DELETE FROM {table} WHERE {where};\n')
```

Comment:
That is the correct way to perform `down` migration. Deleting all elements from the database was not correct, we need to delete only the elements that were inserted by the `up` migration. In addition to that, the original code emptied the last table only, not all tables.